mongofiles/mongofiles.go

```diff
@@ -123,6 +128,14 @@ func (mf *MongoFiles) ValidateCommand(args []string) error {
 		}
 
 		mf.FileNameList = args[1:]
+	case GetRegex:
+		// mongofiles get_regex ... should work over a PCRE
+		// and a string of options passed to the $regex query
+		if len(args) == 1 || args[1] == "" {
+			return fmt.Errorf("'%v' argument missing", args[0])
+		}
+
+		mf.FileNameRegex = args[1]
```

Comment:
We also want a `if len(args) > 2` check. You can copy the check from List.

Reply:
Consider it done.